src/pages/searchOperators.tsx

```diff
@@ -0,0 +1,162 @@
+                                                    className="govuk-checkboxes__input"
+                                                    id={`checkbox-${index}`}
+                                                    name={operatorPublicName}
+                                                    type="checkbox"
+                                                    value={`${nocCode}#${operatorPublicName}`}
+                                                />
+                                                <label
+                                                    className="govuk-label govuk-checkboxes__label"
+                                                    htmlFor={`checkbox-${index}`}
+                                                >
+                                                    {operatorPublicName}
+                                                </label>
+                                            </div>
+                                        );
+                                    })}
+                                </div>
+                                <div className="govuk-!-margin-top-7">
+                                    <input
+                                        type="submit"
+                                        value="Add Operator(s)"
+                                        id="add-operator-button"
+                                        className="govuk-button govuk-button--secondary"
+                                    />
+                                </div>
+                                <div>
+                                    <input
+                                        type="submit"
+                                        value="Continue"
+                                        id="continue-button"
+                                        className="govuk-button"
+                                    />
+                                </div>
+                            </>
+                        </FormElementWrapper>
+                    ) : null}
+                </div>
+            </>
+        </CsrfForm>
+    </TwoThirdsLayout>
+);
+
+export const getServerSideProps = async (ctx: NextPageContextWithSession): Promise<{ props: SearchOperatorProps }> => {
+    const nocCode = getAndValidateNoc(ctx);
+
+    if (!nocCode) {
+        throw new Error('Necessary cookies not found to show operators page');
+    }
+
+    const searchOperatorsAttribute = getSessionAttribute(ctx.req, SEARCH_OPERATOR_ATTRIBUTE);
+
+    const { searchOperator } = ctx.query;
+
+    if (isSearchOperatorAttributeWithErrors(searchOperatorsAttribute) && searchOperatorsAttribute.errors.length > 0) {
+        const { errors } = searchOperatorsAttribute;
+        return {
+            props: {
+                errors,
+                searchText: '',
+                operators: [],
+            },
+        };
+    }
+
+    const searchText = searchOperator && searchOperator !== '' ? searchOperator.toString() : '';
```

Comment:
could we use our `removeExcessWhiteSpace` method from the validator file to help ensure we get only the characters?